src/Collection/Collection.service.ts

```diff
@@ -131,4 +151,70 @@ export class CollectionService {
 
     return true
   }
+
+  public async deleteCollection(collectionId: string): Promise<void> {
+    const collection = await this.getDBCollection(collectionId)
+    if (collection.urn_suffix !== null && collection.third_party_id !== null) {
```

Comment:
We might want to at least add a TODO here, as this check is repeated in `toFullCollection` and on line 195 here. I think it's a good candidate for a `" is third party "` method. In [this PR](https://github.com/decentraland/builder-server/pull/335/files) I renamed the current isTPWCollection to show that the method was using the URN to check, so we have the name available

Reply:
Done. I've added a method in the `CollectionService` class to replace them.